app/pages/sections/order-items/catalogue-solutions/routes.js

```diff
@@ -64,8 +64,8 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
 
   router.get('/:orderItemId', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
     const { orderId, orderItemId } = req.params;
+    const { solutionAlreadySelected } = req.query;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
-
     const pageData = await getOrderItemPageDataBulk({
```

Comment:
See my earlier comment. Rather than relying on a query parameter can we check to see if the catalogue item has already been added to the order here?

Reply:
removed new query parameter and instead checking the already saved catalogue item  and updating the backlink